packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -59,6 +59,45 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       })
     })
 
+    it('can translate, create, and publish a draft of a policy intent', async () => {
+      const draft: NoParkingIntentDraft = {
+        intent_type: 'no_parking',
+        rule_fields: {
+          geographies: [],
+          days: null,
```

Comment:
May also want to provide some placeholder value for days and geographies too just to be able to assert that they were correctly assigned in the policy. Null/empty array might hide bugs that cause only default values to be passed through